crates/indexer/src/models/coin_models/coin_activities.rs

```diff
@@ -153,6 +138,19 @@ impl CoinActivity {
                 } else {
                     (None, None)
                 };
+
+            let maybe_coin_supply = if let APIWriteSetChange::WriteTableItem(table_item) = &wsc {
+                CoinSupply::from_write_table_item(
+                    table_item,
+                    maybe_aptos_coin_info,
+                    txn_version,
+                    txn_timestamp,
+                )
+                .unwrap()
+            } else {
+                None
+            };
```

Comment:
some coins may choose to use integer stored here: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/aggregator/optional_aggregator.move#L63

Reply:
Yea this isn't meant to be a generic supply yet. It only works for aptos coin right now.